index.html

```diff
@@ -21,6 +21,7 @@ <h1 class="d2l-heading-2">Demos</h1>
 
 	<h2 class="d2l-heading-3">Components</h2>
 	<ul>
+		<li><a href="components/typography/demo/typography.html">d2l-typography</a></li>
```

Comment:
We should move this down so it's  in alpha order.

Reply:
Will do